solr/core/src/java/org/apache/solr/response/transform/DocTransformer.java

```diff
@@ -52,6 +53,21 @@ public void setContext( ResultContext context ) {
 
   }
 
+  /**
+   * If this transformer wants to bypass escaping in the {@link org.apache.solr.response.TextResponseWriter} and
+   * write content directly to output for certain field(s), the names of any such field(s) should be returned
+   *
+   * NOTE: normally this will be conditional on the `wt` param in the request, as supplied to the
+   * {@link DocTransformer}'s parent {@link TransformerFactory} at the time of transformer creation.
+   *
+   * @return Collection containing field names to be written raw, or <code>null</code> if no field names should
+   * be written raw. Any collection returned collection need not be externally modifiable -- i.e.,
+   * {@link java.util.Collections#singleton(Object)} is acceptable.
+   */
+  public Collection<String> getRawFields() {
```

Comment:
There doesn't seem to be a semantic difference in null vs an empty collection, and it would be clearer to return non-null so maybe let's just do that?